src/components/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -2,8 +2,10 @@ import { cn } from '@/lib/utils'
 import Lottie from 'lottie-react'
 import loadingAnimation from '@/public/loading.json'
 
-export const LoadingSpinner = ({ className = '' }) => (
+const LoadingSpinner = ({ className = '' }) => (
```

Comment:
Why did you changes these from export const to export default? This creates extra debugging work and possible confusion in the future when someone does something like this:

```tsx
import whateverNameIWant from './folder/'
```

Where as the way itwas, explicitly makes the developer import in the name of the method:
```tsx
import { LoadingSpinner } from './folder/Filename.txs'
```

Reply:
So yea, this is something that needs discussing. Without it we were getting this error.
<img width="512" alt="d8b1dbe6-d847-40c6-9a3d-af74d787b144" src="https://github.com/user-attachments/assets/ede9fa28-a1ef-47cd-830b-c61fb5a692e1">
To resolve it we make it load in the browser with (nextdynamic)[with https://nextjs.org/docs/app/building-your-application/optimizing/lazy-loading#nextdynamic], which expects a default export of a component. This solution was found [here](https://stackoverflow.com/a/77824774).

Reply:
Okay, that works for me for now.

Reply:
Did you try to 'use client' in this component?